packages/mui-material/src/AvatarGroup/AvatarGroup.d.ts

```diff
@@ -27,9 +27,8 @@ export interface AvatarGroupProps extends StandardProps<React.HTMLAttributes<HTM
    * You can override the existing props or add new ones.
    *
    * This prop is an alias for the `slotProps` prop.
-   * It's recommended to use the `slotProps` prop instead, as `componentsProps` will be deprecated in the future.
    *
-   * @default {}
+   * @deprecated use the `slotProps` prop instead. This prop will be removed in v7. [How to migrate](/material-ui/migration/migrating-from-deprecated-apis/).
```

Comment:
May I ask you to update this message as such 😊: https://github.com/mui/material-ui/pull/41875/commits/0f8665c1328cc1b8ac4086b1935dc2f29f4cd3c0#diff-11871663eff3ef5b4247ef4a213a1103e7426bca63d1ac26acef759f045c362a